examples/online_serving/chart-helm/templates/deployment.yaml

```diff
@@ -72,16 +72,21 @@ spec:
         {{ toYaml . | nindent 8 }}
         {{- end }}
 
-      {{-   if .Values.extraInit  }}
+      {{- if or (and .Values.extraInit .Values.extraInit.modelDownload.enabled) .Values.extraInit.initContainers }}
       initContainers:
+      {{- if and .Values.extraInit .Values.extraInit.modelDownload.enabled }}
```

Comment:
Do you still need to check `.Values.extraInit` again or is `.extraInit.modelDownload.enabled` sufficient given the if statement on line 75?